oauth2client/devshell.py

```diff
@@ -117,6 +124,10 @@ def __init__(self, user_agent=None):
     def _refresh(self, http_request):
         self.devshell_response = _SendRecv()
         self.access_token = self.devshell_response.access_token
+        expires_in = self.devshell_response.expires_in
+        if expires_in is not None:
```

Comment:
Can `self.devshell_response.expires_in` be `None` at this point? Can `self.token_expiry` be non-`None` at this point? If both are possible, is this method correct in the way it would leave the previous non-`None` value unmolested?